src/main/java/bruh/task/Task.java

```diff
@@ -0,0 +1,85 @@
+package bruh.task;
+
+import bruh.exception.BruhException;
+import bruh.exception.InvalidArgumentException;
+
+import java.io.Serializable;
+
+/**
+ * Represents a task that can be mark as done.
+ */
+public abstract class Task implements Serializable {
+    private final String description;
+    private final char symbol;
```

Comment:
I believe constant variable should be in all uppercase?

Reply:
Thanks for pointing this out! For these variables, they are not exactly constants; I added the `final` keyword for immutability, to reduce complexity and hopefully catch bugs, so I'm not too sure if these should be in all uppercase.